client/constants.js

```diff
@@ -1,3 +1,5 @@
+// TODO Organize this file by reducer type, ot break this apart into
+// multiple files
```

Comment:
👍  I think having different files would be good as I struggled a bit figuring out where new action constants should go.